openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -52,10 +52,13 @@ export class ReadMoreComponent {
     }
 
     reset() {
-        this.fullHeight = this.$content.scrollHeight;
+        const fullHeight = this.$content.scrollHeight;
```

Comment:
This should still be `this.fullHeight`; the `reset` method runs on resize and updates all the relevant size parameters, since the height can change during a resize.

```suggestion
        this.fullHeight = this.$content.scrollHeight;
```